Tella/Components/Tabs/Pages.swift

```diff
@@ -6,13 +6,16 @@ import Foundation
 
 public enum Pages: Hashable {
     
+    case templates
     case draft
     case outbox
     case submitted
     
     var title: String {
         switch self {
             
+        case .templates:
```

Comment:
https://github.com/Horizontal-org/Tella-iOS/pull/74#discussion_r1338550736
If you can localise the strings for Uwazi and  Reports titles